packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -1915,7 +1915,9 @@ export class ComboBox extends BaseComponent<IComboBoxProps, IComboBoxState> {
    * Click handler for the autofill.
    */
   private _onAutofillClick = (): void => {
-    if (this.props.allowFreeform) {
+    const { disabled } = this.props;
+
+    if (this.props.allowFreeform && !disabled) {
```

Comment:
Nit: This should be consistent and either we deconstruct into `const {disabled, allowFreefrom} = this.props` or the if statement is like `if (this.props.allowFreefrom && !this.props.disabled)`

Reply:
I agree. I have adjusted to be consistent.